src/client/tile.cpp

```diff
@@ -2048,3 +2051,36 @@ video::SColor TextureSource::getTextureAverageColor(const std::string &name)
 	c.setAlpha(255);
 	return c;
 }
+
+video::ITexture *TextureSource::getShaderFlagsTexture(TileDef *tiledef, TileSpec *tile)
+{
+	video::ITexture *flags_texture;
+	std::string tname = "__shaderFlagsTexture";
+	std::string strue = "1";
+	std::string sfalse = "0";
+
+	bool normalmap_present = tile->normal_texture ? true : false;
+	tname += normalmap_present ? strue : sfalse;
+	tname += tiledef->tileable_horizontal ? strue : sfalse;
```

Comment:
WTF is up with this "strue" and "sfalse"?  Why not just make it "1" : "0" instead?